stdlib/kokorawr/kokorawr/functions/services/rps.js

```
@@ -1,8 +1,4 @@
-const scores = require('./scores.js');
-
-let waiting = [];
-let waitingTeam = -1;
-let options = ['rock', 'paper', 'scissors']
+const lib = require('lib')({token: process.env.STDLIB_TOKEN});
 
 /**
 * Rock Paper Scissors!
```

This should throw an error

```
$ lib .commands.rps dude derp state
{
  "response_type": "in_channel",
  "text": "(Rock Paper Scissors) <@dude>:Red is waiting for a match..."
}
```